### ISSUE TITLE
403 forbidden link as "Learn more about MAAS"

### ISSUE THREAD

**Issue by nobuto-m** [canonical/ubuntu.com#12632]
There is a link to an external website as:
https://technology.amis.nl/platform/kubernetes/charmed-kubernetes-on-kvm-using-maas-and-juju/

https://github.com/canonical/ubuntu.com/blob/14cd524a864d1375510b33e2a14b6b09340110d7/templates/kubernetes/charmed-k8s.html#L198

The site somehow returns 403 Forbidden when it's accessed from Firefox on Ubuntu (curl, wget, and Chromium are just fine). I understand that's not an issue of ubuntu.com itself but it would be good to be replaced with something in the site itself.

---
*Reported from: https://ubuntu.com/kubernetes/charmed-k8s*

**Comment by nobuto-m** [comment]
> @nobuto-m do you still see a 403?

Yes, with a real browser. You can also confirm it with the curl commands above.

**Comment by tbille** [comment]
oh I see. Weird. Will ask around to see with what link it can be replaced. Thanks for reporting :)

**Comment by britneywwc** [comment]
Thanks for reporting @nobuto-m, I've reached out to the PM to see if we could replace it with an internal link. Will keep you posted.

**Comment by britneywwc** [comment]
This should be fixed now, I have replaced the link with an internal link. I will be closing this issue, feel free to reopen if the issue persists. Thanks!
